internal/flags/flags.go

```diff
@@ -115,4 +115,7 @@ const (
 	Normalization                   = "normalization"                   // Normalization flag
 	Backwards                       = "backwards"                       // Backwards flag
 	Strength                        = "strength"                        // Strength flag
+	LogTypes                        = "logTypes"                        // LogTypes flag
+	SizeRequestedPerFileBytes       = "sizeRequestedPerFileBytes"       //SizeRequestedPerFileBytes flag
+	LogRedacted                     = "redacted"                        // LogRedacted flag
```

Comment:
This could have stayed `Redacted`, there's no log in the value so anyone should be free to use it